Cleanup.ps1

```diff
@@ -1,4 +1,4 @@
-param([PSCredential]$Credential, [string]$TenantId)
+param([Parameter(Mandatory=$false)][PSCredential]$Credential=$null, [Parameter(Mandatory=$true)][string]$TenantId)
```

Comment:
Made the credentials optional and the tenant id mandatory.
Having the tenant id as mandatory is much simpler for the end-user and fits better with the instructions. If they don't supply the tenant id on the command line then they will be prompted for it automatically.